graviteeio/config.yml

```diff
@@ -0,0 +1,7 @@
+name: Gravitee.io
```

Comment:
Could you please add APIM tor the name ? 
We could work on other products.

Gravitee.io APIM